manifests/install.pp

```diff
@@ -55,5 +58,13 @@
       tag    => 'nodesource_repo',
     }
   }
-}
 
+  file { 'root_npmrc':
```

Comment:
I think we should do what's in the PR :D

or file_line if you'd like.. whatever get's this merged.